doc/book/app_server/luajit_getmetrics.rst

```diff
@@ -209,9 +209,9 @@ allocated somewhere.
 It is a good thing if the slope curve of
 ``strhash_miss`` is less than the slope curve of ``strhash_hit``.
 
-The other gc_*num values -- ``gc_cdatanum``, ``gc_tabnum``, ``gc_udatanum`` -- can be accessed
+The other ``gc_*num`` values -- ``gc_cdatanum``, ``gc_tabnum``, ``gc_udatanum`` -- can be accessed
 in a similar way.
-Any of the gc_*num values can be useful when looking for memory leaks – the total 
+Any of the ``gc_*num`` values can be useful when looking for memory leaks – the total 
```

Comment:
The commit concerns formatting errors like this one:
![image](https://user-images.githubusercontent.com/40490252/142394671-6a450657-8868-4cb8-8808-9255f43be174.png)